Updating readme for clarity (#23)
sensu/sensu-slack-handler: README.md

[![Bonsai Asset Badge](https://img.shields.io/badge/Sensu%20Slack%20Handler-Download%20Me-brightgreen.svg?colorB=89C967&logo=sensu)](https://bonsai.sensu.io/assets/sensu/sensu-slack-handler) [![Build Status](https://travis-ci.org/sensu/sensu-slack-handler.svg?branch=master)](https://travis-ci.org/sensu/sensu-slack-handler)

# Sensu Go Slack Handler

The Sensu Slack handler is a [Sensu Event Handler][1] that sends event data to
a configured Slack channel.

## Configuration

### Asset registration

Assets are the best way to make use of this handler. If you're not using an asset, please consider doing so! If you're using sensuctl 5.13 or later, you can use the following command to add the asset: 

> `sensuctl asset add sensu/sensu-slack-handler:1.0.3`

Reviewer: I know I gave an example with the version included, but I think leaving it out will keep things low-touch and avoid the perception that the readme is out of date.

```suggestion
`sensuctl asset add sensu/sensu-slack-handler`
```

> If you're using an earlier version of sensuctl, you can find the asset on the [Bonsai Asset Index](https://bonsai.sensu.io/assets/sensu/sensu-slack-handler).

Reviewer: ```suggestion
If you're using an earlier version of sensuctl, you can download the asset definition from [this project's Bonsai Asset Index page](https://bonsai.sensu.io/assets/sensu/sensu-slack-handler).
```

        "timeout": 30,
        "filters": [
            "is_incident"
        ],

>         "runtime_assets": ["sensu-slack-handler_linux_amd64"]

Reviewer: Let's check that `asset add` produces an asset name like this one... I hope it is installing multi-build asset definitions with simpler names...
